plane/source/docs/guide-tilt-rotor.rst

```diff
@@ -66,9 +65,11 @@ hovering. Currently supported tilt-rotor frame classes are:
    <tr><td>Tricopter</td><td>7</td></tr>
    </table>
 
+.. note: the BiCopter tilt-rotor (either motors tilting ,or entire wing with fixed motors tilting) QuadPlane is a special case. It requires that :ref:`Q_FRAME_CLASS<Q_FRAME_CLASS>` be set to 10 which is reserved for Tailsitter configurations only, normally. See :ref:`BiCopter <bicopter>`  below.
```

Comment:
```suggestion
.. note: the BiCopter tilt-rotor QuadPlane is a special case. It requires that :ref:`Q_FRAME_CLASS<Q_FRAME_CLASS>` be set to 10 which is reserved for Tailsitter configurations only, normally. See :ref:`BiCopter <bicopter>`  below.
```

Bi-copter needs its two motors to be able to tilt interdependently.

Reply:
fixed